stable/fairwinds-insights/ci/test-values.yaml

```diff
@@ -74,6 +74,10 @@ databaseCleanupCronjob:
   enabled: true
   schedules: []
 
+resourcesRecommendationsCronjob:
+  enabled: false
+  schedules: []
```

Comment:
I think we should try and enable it during testing, so we catch any issues with the YAML

Reply:
Ditto for other cronjobs, but we don't have to add that now

Reply:
oh that's right